lib/src/theme/flutter_deck_text_theme.dart

```diff
@@ -4,6 +4,10 @@ import 'package:flutter/widgets.dart';
 class FlutterDeckTextTheme {
   /// Creates a theme to style text in a slide deck.
   const FlutterDeckTextTheme({
+    this.display = const TextStyle(
+      fontSize: 91,
```

Comment:
Sorry, my bad, but after opening the slide, this font size feels a little bit too small. Let's set it to 103.